spec/requests/get_involved_spec.rb

```diff
@@ -15,6 +16,29 @@
   end
 end
 
+RSpec.describe "Time until closure" do
```

Comment:
I don't think we've documented this well, but the convention we have is for the top-level descriptor to use `RSpec.describe` and all of the other blocks to in the test file to either use `describe` if describing a method name, or `context` if describing a scenario. I think for this test only the "when the consultation is closed" context is needed.

Reply:
Good spot! Thanks